apps/web/components/eventtype/EventSetupTab.tsx

```diff
@@ -30,6 +31,34 @@ export const EventSetupTab = (
   const [showLocationModal, setShowLocationModal] = useState(false);
   const [editingLocationType, setEditingLocationType] = useState<string>("");
   const [selectedLocation, setSelectedLocation] = useState<OptionTypeBase | undefined>(undefined);
+  const [multipleDuration, setMultipleDuration] = useState(eventType.metadata.multipleDuration);
+
+  const multipleDurationOptions = [
+    { value: 5, label: t("multiple_duration_mins", { count: 5 }) },
+    { value: 10, label: t("multiple_duration_mins", { count: 10 }) },
+    { value: 15, label: t("multiple_duration_mins", { count: 15 }) },
+    { value: 20, label: t("multiple_duration_mins", { count: 20 }) },
+    { value: 25, label: t("multiple_duration_mins", { count: 25 }) },
+    { value: 30, label: t("multiple_duration_mins", { count: 30 }) },
+    { value: 45, label: t("multiple_duration_mins", { count: 45 }) },
+    { value: 50, label: t("multiple_duration_mins", { count: 50 }) },
+    { value: 60, label: t("multiple_duration_hour", { count: 60 }) },
+    { value: 75, label: t("multiple_duration_hour_mins", { count: 75, mins: 15 }) },
```

Comment:
Quite cool that you can combine translation keys within translation keys, otherwise these translations would be crazy.